site/gatsby-site/src/components/submissions/SubmissionForm.js

```diff
@@ -273,7 +274,24 @@ const SubmissionForm = () => {
         </Form.Group>
 
         <Form.Group className="mt-3">
-          <Label popover={POP_OVERS['tags']} label={'Tags'} />
+          <Label popover={POP_OVERS['language']} label={'Language'} />
+          <Form.Select
+            name="language"
+            placeholder="Report Language"
+            className="mt-3"
```

Comment:
Using `mt-3` here makes the space between the label and the input larger than it is for the other form entries.

![image](https://user-images.githubusercontent.com/25443411/177784275-dab02a68-15d6-4cbe-a07b-68be68c0533e.png)